pkg/registry/scripts/projects.js

```diff
@@ -128,7 +167,13 @@
                 var display = fields.display.trim();
                 var description = fields.description.trim();
 
-                if (!NAME_RE.test(name)) {
+                if (!name) {
```

Comment:
wanted to show a "Name cannot be empty" error message
to show that it is a mandatory field